features/networking/multus.feature

```diff
@@ -991,3 +991,36 @@ Feature: Multus-CNI related scenarios
     When I execute on the pod:
       | /usr/sbin/ip | a |
     Then the output should contain "192.168.1"
+
+  # @author weliang@redhat.com
+  # @case_id OCP-25910
+  @admin
+  Scenario: Assign static MAC address using pod annotation				 
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/runtimeconfig-def-mac.yaml |
+      | n | <%= project.name %>                                                                                               |
+    Then the step should succeed
+    
+    # Create a pod absorbing above net-attach-def
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/runtimeconfig-pod-mac.yaml |
```

Comment:
please try to avoid using personal github account for test yaml files.  Use the [v3-testfiles](https://github.com/openshift-qe/v3-testfiles) repo instead.

Reply:
Wait for openshift-qe/v3-testfiles#1206

Reply:
Use /openshift-qe/v3-testfiles public repository now